docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
 > com.r3.conclave.common.InvalidEnclaveException: Enclave code signer does not match any of the acceptable code signers.
 (key hash 0000000000000000000000000000000000000000000000000000000000000000 vs acceptable
 2222222222222222222222222222222222222222222222222222222222222222)
 
-For more information on how to choose a constraint, see [Enclave constraints](constraints.md).
-
+For more information on choosing a constraint, see [Enclave constraints](constraints.md).
 
 ## Beyond mock mode
-Eventually you will want to run your application in one of the other [enclave modes](enclave-modes.md), namely simulation,
-debug, or release. This section describes how to build and run the sample in these other modes.
+This section describes how to build and run the sample in other [enclave modes](enclave-modes.md), namely
+simulation, debug, and release.
 
-**First, make sure that your system fulfills the
-[system requirements](enclave-modes.md#system-requirements) for your desired mode.**
+*First, ensure that your system fulfills the [system requirements](enclave-modes.md#system-requirements) for
+your desired mode.*
 
 ### Build the enclave in other modes
 
-The hello world sample has been [configured](enclave-modes.md#set-the-enclave-mode) such that you can supply the mode
-via the `enclaveMode` Gradle parameter. For example, you could compile the host for simulation mode with the command
+The hello world sample has been [configured](enclave-modes.md#set-the-enclave-mode) such that you can define the mode
+using the `enclaveMode` Gradle parameter. For example, you can compile the host for simulation mode with the command
+
 === "Windows"
-    ```bash
+```bash
     gradlew.bat :host:bootJar -PenclaveMode=simulation
-    ```
+```
 === "macOS / Linux"
-    ```bash
+```bash
     ./gradlew :host:bootJar -PenclaveMode=simulation
-    ```
+```
+
+This generates the host JAR at `host/build/libs/host-simulation.jar`.
 
-This would generate the host JAR at `host/build/libs/host-simulation.jar`.
+!!!Note
 
-!!! tip
-    If working from inside IntelliJ, find the bootJar task of your host project from the Gradle menu on the right-hand 
-    side. Modify run configuration of this task by adding the -PenclaveMode=<MODE> flag to the arguments section.
+    If you use IntelliJ, find the bootJar task of your host project from the Gradle menu on the right-hand 
+    side. Modify the run configuration of this task by adding the -PenclaveMode=<MODE> flag to the arguments section.
     Now you can run the task.
 
-For **release mode**, the sample app has been configured (in the `build.gradle` of the `enclave` subproject) to use
-external signing. Note that **external signing is optional** and you do not have to include it in your own projects.
-See [Enclave signing](signing.md) for more information on external signing.
+For *release mode*, the sample app is configured (in the `build.gradle` of the `enclave` subproject) to use
```

Comment:
Perhaps "external code signing" to more clear, in line with my previous comment.

To give a quick bit of background. There's two types of "signing" that's linked to an enclave:
* This "code signer" or "code signing", which is the enclave's binary file (which is loaded by the host application) being signed by an external RSA key.
* Inside the enclave code, the enclave has the ability to sign arbitrary data bytes for the user. This is what what "signing" would probably normally mean to a developer.

Reply:
I would highlight more release mode, maybe even as a new sub heading

Reply:
Added an H4 subheading.